baseband/mark4/tests/test_mark4.py

```diff
@@ -605,15 +605,15 @@ def test_filestreamer(self, tmpdir):
 
         # Test that squeeze attribute works on read (including in-place read)
         # and write, but can be turned off if needed.
-        with mark4.open(SAMPLE_FILE, 'rs', ntrack=64, decade=2010) as fh:
-            assert fh.sample_shape == (8,)
-            assert fh.sample_shape.nchan == 8
-            assert fh.read(1).shape == (8,)
+        with mark4.open(SAMPLE_FILE, 'rs', ntrack=64,
+                        thread_ids=[0], decade=2010) as fh:
```

Comment:
Adding `thread_ids` is fine on its own, but I have more extensive subsetting tests in #118.  Perhaps keep the original test?  (If it's good practice for each PR to be self-contained, I understand keeping it.)